cmd/swarmctl/network/common.go

```diff
@@ -32,6 +32,12 @@ func GetNetwork(ctx context.Context, c api.ControlClient, input string) (*api.Ne
 		}
 
 		if l := len(rl.Networks); l > 1 {
+			for _, n := range rl.Networks {
+				if n.Spec.Annotations.Name == input {
+					// Found a full name match
```

Comment:
This needs to handle the case where there may be two matches. There should only be one exact match.

This code should not assume uniqueness, as that is a server-side policy.